Dockerfile

```diff
@@ -31,6 +31,14 @@ RUN apt update -y -q && apt upgrade -y -q && apt update -y -q && \
     lldb \
     gettext \
     ninja-build \
+    file \
+    perl \
+    xxd \
+    mesa-common-dev \
+    zlib1g-dev \
+    libelf-dev \
+    libdrm-dev \
+    libudev-dev \
```

Comment:
The need for these build dependencies is the only reason why I didn't just add an `install_hip()` to https://github.com/compiler-explorer/infra/blob/main/update_compilers/install_libraries.sh

Though, I suppose that if I built them from source, I could just this PR and do everything in infra.

Reply:
Ah. The install_libraries.sh script is deprecated. So, the misc-builder is the way to go.